app/controllers/boxes_controller.rb

```diff
@@ -96,12 +108,38 @@ def load_box
     @box = Box.where(institution: @navigation_context.institution).find(params.fetch(:id))
   end
 
+  def load_box_print
+    begin
+      @box = Box.where(institution: @navigation_context.institution).find(params.fetch(:id))
+    rescue ActiveRecord::RecordNotFound
+      @boxes = TransferPackage
+        .within( @navigation_context.institution )
+        .find( params.fetch( :transfer_package ) )
```

Comment:
There might be a deeper authorization problem here, though.

Consider that a box that has been transferred multiple times:

* Institution I creates box B
* Institution I transfers box B to institution J
* Institution J transfers box B to institution K

Should institution I still be able to access the inventory of box B, even after it has transferred to J and then K?

/cc @diegoliberman

Reply:
This ticket changed over the time I was doing. The transfer package was needed considering the option to send a box multiple times, that's why I needed this parameter. But at the end, we decided to don't permit this anymore. So yes, probably, we don't need this logic anymore.

Reply:
I don't know if it's a good idea to remove transfer_package now. It's not a thing the will cause some problem, and it will be easier to make a multiple transfer functionality in the future. In this specific lines of code, I removed this param and got errors, so if I remove it, I will need to rebuild and test again the entire flow of the task. This `raise` treats the cases that the the box is not related to any institution, so I need to get the transfer. I changed this query a lot, I was doing it through BoxTransfers Models too, but this one is the sorter option.